### PR TITLE
DOCS-602: Remove EXIF data

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,21 @@ To ensure your markdown is properly formatted, run `make markdowntest`.
 
 To check for broken links run `make htmltest`.
 
+## Linters and Checks
+
+To ensure that you do not accidentally add `EXIF` data on images, please add the following lines to your `.git/hooks/pre-commit` file:
```
Comment (andf-viam) [inline README.md:52]:
```suggestion
To ensure that you do not accidentally add `EXIF` data on images, please add the following lines to your `.git/hooks/pre-commit` file. If you do not have a file of that name in that location, create one, or rename the existing `pre-commit.sample` file in that directory as `pre-commit`, and add the code below:
```

**Review by andf-viam — APPROVED** [review]
LGTM! Thanks for doing this!!

File: README.md (hunk 1 above)
Comment (andf-viam) [reply README.md:52]: Worth noting that `.git` is explicitly excluded in our git repo config, so this file is not part of our version control workflow. i.e. You gotta add this to your local clone of your fork by hand.

Comment (npentrel) [reply README.md:52]: It's not really explicitly excluded. You can't manage the .git folder in git in any project. Happy to add clarification though.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,21 @@ To ensure your markdown is properly formatted, run `make markdowntest`.
 
 To check for broken links run `make htmltest`.
 
+## Linters and Checks
+
+To ensure that you do not accidentally add `EXIF` data on images, please add the following lines to your `.git/hooks/pre-commit` file. If you do not have a file of that name in that location, create one, or rename the existing `pre-commit.sample` file in that directory as `pre-commit`, and add the code below:
```
Comment (npentrel) [inline README.md:52]:
```suggestion
To ensure that you do not accidentally add `EXIF` data on images, please install [exiftool](https://exiftool.org/install.html) and add the following lines to the `.git/hooks/pre-commit` file in your local files.
If you do not have a file of that name in that location, create one, or rename the existing `pre-commit.sample` file in that directory as `pre-commit`, and add the code below:
```

**Pull request merged**